.changes/3.38.2

```diff
@@ -1,5 +1,9 @@
 [
     {
+        "type": "enhancement",
```

Comment:
This update needs to go in its own file, not the previous merged change blob.

>A changelog document is a small JSON blob placed in the .changes/nextrelease folder. It should be named a clearly and uniquely, akin to a branch name.

You can find a sample [in this PR](https://github.com/aws/aws-sdk-php/pull/1376/files#diff-058c075a21877f6b3127b93c0b89bf32).

Nit since this has to move: the `"category"` should be the singular `"Test"` to match the namespaces for those files.

Reply:
I guess now everything is right. Sorry for the inconvenient.